examples/autocomplete.html

```diff
@@ -217,6 +217,10 @@
               ]);
             },
 
+          },
+          tagsModule: {
```

Comment:
Well Bloodhound does give enough flexibility to handle remote calls from a json file. And you can see I am not directly passing parameters to the Bloodhound engine rather a module property with the same name as Bloodhound property.